src/components/list/LibraryList.vue

```diff
@@ -17,34 +17,45 @@
       !isSelected ? 'scroll' : '',
     ]"
   >
+
     <template
       v-for="(drawerItem) in drawerItems"
       :key="drawerItem.id"
     >
       <library-item
-        v-if="drawerItem.size > 0 && (selectedItemId === drawerItem.id || selectedItemId === null)"
+        v-if="drawerItem.size > 0
+          && (selectedItemId === drawerItem.id || selectedItemId === null)"
         :item="drawerItem"
         :is-selected="isSelected"
         @click="onItemClick(drawerItem.id)"
       />
     </template>
-    <q-item
-      v-if="selectedItemId && !isEmptyList"
-      style="overflow-y: auto !important;"
-      class="q-pa-none q-mb-sm"
-    >
-      <component-definition-grid
-        :definitions="selectedItemDefinitions"
-        :plugin-name="plugin.data.name"
-        class="q-mt-xs q-pr-sm"
-      />
-    </q-item>
+    <template v-if="$acl.role('create-component')">
+      <q-item
+        v-if="selectedItemId && !isEmptyList"
+        style="overflow-y: auto !important;"
```

Comment:
in `<style>`

Reply:
rework on all code, but don't understand your comment.